target/linux/generic/config-5.4

```diff
@@ -1161,7 +1161,7 @@ CONFIG_DEBUG_FS=y
 # CONFIG_DEBUG_INFO is not set
 # CONFIG_DEBUG_INFO_BTF is not set
 # CONFIG_DEBUG_INFO_DWARF4 is not set
-CONFIG_DEBUG_INFO_REDUCED=y
+# CONFIG_DEBUG_INFO_REDUCED is not set
```

Comment:
Shouldn't this be added to `config-5.10` too?

Reply:
Yes, thanks for noting. The patch set is 8 months old and I see things have changed underneath it.

Will update that, and also some archs that have since added `CONFIG_DEBUG_INFO=y`.

Reply:
Thank you.